develop-docs/ingestion/relay/relay-best-practices.mdx

```diff
@@ -16,6 +16,18 @@ Make sure changes to the event protocol and APIs are forward-compatible. Relay s
 or truncate data it does not understand. It is a supported use-case to have customers running
 outdated Relays but up-to-date SDKs.
 
+For example, `enum`s in the event protocol (or any other data type received from the SDK) need to have a catchall variant
+such that customer relays running an older version can still forward the data:
+
+```rust
+enum EventColor {
+    Red,
+    Green,
+    Blue,
+    /// Unknown color, for forward compatibility.
+    Unknown(String),
```

Comment:
Sometimes we use `#[serde(other)] Unknown`, sometimes `Unknown(String)` and other times our derives have something builtin `#[metastructure(fallback_variant)]`.

Seems to be quite use-case specific and hard to phrase :/.